docs/website/data/integrations.yaml

```diff
@@ -1483,6 +1509,15 @@ organizations:
   pulumi:
     name: Pulumi
     link: https://www.pulumi.com/
+  walt.id:
```

Comment:
I think this should be `waltid` to match [this line](https://github.com/open-policy-agent/opa/pull/4841/files#diff-e229f15367af01d0a2953d313a8ffe754ad0b4a8d00b6088cf355343bc13a4b1R1353) and to unbreak the website preview for this PR.